<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="6" total="10" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-30T17:13:51Z" name="Single" finished-at="2023-04-30T17:14:27Z" duration-ms="35997">
    <groups>
      <group name="Sanity">
        <method signature="TestTO.verifyFormCanSubmitSuccssfully()[pri:2, instance:eat.tests.TestTO@17f7cd29]" name="verifyFormCanSubmitSuccssfully" class="eat.tests.TestTO"/>
        <method signature="TestTO.verifyIfContactUsPageIsLoading()[pri:6, instance:eat.tests.TestTO@17f7cd29]" name="verifyIfContactUsPageIsLoading" class="eat.tests.TestTO"/>
        <method signature="TestTO.verifyIfRequestPageIsLoading()[pri:9, instance:eat.tests.TestTO@17f7cd29]" name="verifyIfRequestPageIsLoading" class="eat.tests.TestTO"/>
        <method signature="TestTO.verifyIfTheApplicationHasLaunch()[pri:1, instance:eat.tests.TestTO@17f7cd29]" name="verifyIfTheApplicationHasLaunch" class="eat.tests.TestTO"/>
      </group> <!-- Sanity -->
    </groups>
    <test started-at="2023-04-30T17:13:51Z" name="SingleTest" finished-at="2023-04-30T17:14:27Z" duration-ms="35997">
      <class name="eat.tests.TestTO">
        <test-method is-config="true" signature="tearUp()[pri:0, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:13:51Z" name="tearUp" finished-at="2023-04-30T17:14:21Z" duration-ms="30153" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearUp -->
        <test-method signature="verifyIfTheApplicationHasLaunch()[pri:1, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:14:21Z" name="verifyIfTheApplicationHasLaunch" finished-at="2023-04-30T17:14:22Z" duration-ms="1597" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfTheApplicationHasLaunch -->
        <test-method signature="verifyFormCanSubmitSuccssfully()[pri:2, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:14:22Z" name="verifyFormCanSubmitSuccssfully" finished-at="2023-04-30T17:14:26Z" duration-ms="4124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFormCanSubmitSuccssfully -->
        <test-method signature="verifyIfContactUsPageIsLoading()[pri:6, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:14:26Z" name="verifyIfContactUsPageIsLoading" finished-at="2023-04-30T17:14:26Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfContactUsPageIsLoading -->
        <test-method signature="verifyIfRequestPageIsLoading()[pri:9, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:14:26Z" name="verifyIfRequestPageIsLoading" finished-at="2023-04-30T17:14:26Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at eat.tests.TestTO.verifyIfRequestPageIsLoading(TestTO.java:97)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfRequestPageIsLoading -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:eat.tests.TestTO@17f7cd29]" started-at="2023-04-30T17:14:26Z" name="tearDown" finished-at="2023-04-30T17:14:27Z" duration-ms="107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- eat.tests.TestTO -->
    </test> <!-- SingleTest -->
  </suite> <!-- Single -->
</testng-results>
